spec/features/issues_spec.rb

```diff
@@ -65,7 +65,7 @@
 
       click_button "Save changes"
 
-      page.should have_content "Assignee: Select assignee"
+      page.should have_content "Assignee: none"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.